src/api.js

```diff
@@ -912,17 +914,6 @@
       || connectionType === connect.ConnectionType.MONITORING;
   }
 
-  /**
-   * Utility method for checking whether this connection is an agent-side connection 
-   * (type AGENT or MONITORING)
-   * @return {boolean} True if this connection is an agent-side connection. False otherwise.
-   */
-  Connection.prototype._isAgentConnectionType = function () {
```

Comment:
Let's not remove this until the next major version bump. Mark it as deprecated and add a warning log